src/mahoji/commands/tames.ts

```diff
@@ -1376,6 +1377,31 @@ async function superGlassCommand(user: MUser, channelID: string) {
 	});
 }
 
+async function superheatItemCommand(user: MUser, channelID: string, itemName: string) {
+	const item = Bars.find(i => getOSItem(i.id).name === itemName);
+	const hasKlik = new Bank((await getUsersTame(user))?.tame?.fed_items as ItemBank).has('Klik');
+
```

Comment:
Something like this might be better:
```ts
const { tame, activity } = user.fetchActiveTame();
if (!tame) return `You don't have a tame selected.`;
if (activity) return `${tame} is already on a trip.`;
const hasKlik = tame.hasBeenFed('Klik');
```